hudi-client/hudi-client-common/src/main/java/org/apache/hudi/io/HoodieMergeHandle.java

```diff
@@ -270,7 +269,7 @@ protected void init(String fileId, Iterator<HoodieRecord<T>> newRecordsItr) {
       // update the new location of the record, so we know where to find it next
       if (needsUpdateLocation()) {
         record.unseal();
-        record.setNewLocation(new HoodieRecordLocation(instantTime, fileId));
+        record.setNewLocation(newRecordLocation);
```

Comment:
the `fileId` input to the method is unused, should it be removed now?

Reply:
sg